core/jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.ts

```diff
@@ -483,7 +483,7 @@ export class CreateServiceComponent implements OnInit {
     //   payload["accounts"]=this.selectedAccount;
```

Comment:
remove?

Reply:
This is on an internal file and is currently irrelevant to oss Jazz

Reply:
well still we have some commented code and it shld be removed even if its internal. We should not treat internal or oss any different.

Reply:
removing commented code